packages/permission_handler/example/tizen/tizen-manifest.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest package="org.tizen.permission_handler_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
-    <profile name="common"/>
-    <ui-application appid="org.tizen.permission_handler_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="4" launch_mode="single">
+    <profile name="wearable"/>
```

Comment:
Is this intended?

Reply:
Yes :) https://github.com/flutter-tizen/plugins/pull/281#discussion_r760807695

Reply:
OMG! Everyone thinks the same way! :)

Reply:
FYI, there's also a written guideline for this. :)

https://github.com/flutter-tizen/flutter-tizen/wiki/New-plugin-submission-guideline#tizen-manifestxml